src/components/date-time/date-time.component.ts

```diff
@@ -22,7 +22,125 @@ const DATE_TIME_VALUE_ACCESSOR = {
 @Component({
   selector: 'swui-date-time',
   providers: [DATE_TIME_VALUE_ACCESSOR],
-  template
+  template: `<div class="swui-date-time">
```

Comment:
ugh ... I suppose thats why they have that custom dealie in the component defs :(